containers/DvmState.ts

```diff
@@ -39,25 +36,29 @@ const useContractState = () => {
   // get state from EMP
   const queryState = async () => {
     if (
-      dvm !== null &&
+      votingContract !== null &&
+      storeContract !== null &&
+      collAddress !== null &&
       priceIdentifier !== null &&
       expirationTimestamp !== null
     ) {
       const res = await Promise.all([
-        dvm.hasPrice(
+        votingContract.hasPrice(
           priceIdentifier.toString(),
           expirationTimestamp.toNumber(),
           { from: empAddress }
         ),
+        storeContract.computeFinalFee(collAddress, { from: empAddress }),
```

Comment:
nit: Does this have to be called from `empAddress`to succeed?

Reply:
nope, does not. good catch. have removed